example/CMakeLists.txt

```diff
@@ -1,11 +1,5 @@
 # Find Kokkos
-find_package(Kokkos QUIET)
-if(NOT Kokkos_FOUND)
-  message(FATAL_ERROR "Kokkos not found, set Kokkos_ROOT properly (current Kokkos_ROOT=${Kokkos_ROOT})")
-endif()
-foreach(_i "1;2;3") # cut .../lib/cmake/Kokkos suffix
-cmake_path(GET Kokkos_DIR PARENT_PATH Kokkos_DIR)
-endforeach()
```

Comment:
This should have been
```
oreach(_i 1;2;3) # cut .../lib/cmake/Kokkos suffix
cmake_path(GET Kokkos_DIR PARENT_PATH Kokkos_DIR)
endforeach()
```
anyway. Note that the intention was to print the `Kokkos` root directory but we are now printing the path to the directory holding the configuration.

Reply:
See the description.  I understand what the intent was and I don't see any compelling reason to do so.